### ISSUE TITLE
How does it handle object changes Salesforce side?

### ISSUE THREAD

**Issue by riekusr** [subzerocloud/pgsalesforce_demo#1]
Hi, love your project and if it matures a bit i have plenty of usecases. How do you plan to handle changes in the datamodel on the Salesforce side?

**Comment by ruslantalpa** [comment]
There are multiple avenues (though i can't say i necessarily have plans in implementing the advanced ways yet).

- If the data in SF is not huge (below millions of rows/table) i think the best option is just to spin up a fresh db with the new list of objects. this can be reasonably fast if usage of bulk api (or sf backup file) is implemented
- manually use primitives like `pgsalesforce.create_table_pairs` to add new objects on live running systems
- theoretically possible to have a background process monitoring the SF metadata and diffing the jsons and based on those, execute DDLs (like in the second option) to sync the schema layout. I am guessing "new column/new object" are the most common ones and those are easy to implement.

I'm curious about the use cases, I also thought this would be a compelling tool/product (build apps on top of PG, ETLs to PG or even other destinations for which pg has fdws) but so far in my reach-outs the reaction/excitement was "meah" :).
Orgs that don't have engineering departments can't really use it and the ones that do prefer to build their ETLs themselves (mulesoft/appsync/custom)

**Comment by riekusr** [comment]
I agree, at the moment I use ETL via nodered as well, but having something robust in PG seems natural.
